server/collab/matchers/hist_matcher.py

```diff
@@ -14,43 +13,30 @@
 class HistogramMatcher(matcher.Matcher):
   @staticmethod
   def match(source, target):
-    start = time.time()
```

Comment:
Per-matcher timers are available through celery console output, see for example:

    [2017-06-13 20:26:55,099: INFO/MainProcess] Received task: collab.tasks.match[5f7446ed-667f-44a0-8363-6a1a88cfc3d2]
    [2017-06-13 20:26:55,115: WARNING/Worker-5] (2L, 2L, None, None, 2L, None, None, u'["identity_hash", "name_hash", "assembly_hash"]')
    [2017-06-13 20:26:55,131: WARNING/Worker-5] Running task 5f7446ed-667f-44a0-8363-6a1a88cfc3d2
    [2017-06-13 20:26:55,781: WARNING/Worker-5] Matching 2075 local vectors to 2064 remote vectors by collab.matchers.identity_hash.IdentityHashMatcher yielded 30471 matches
    [2017-06-13 20:26:58,133: WARNING/Worker-5] Took: 0:00:03.001320
    [2017-06-13 20:26:58,245: WARNING/Worker-5] Matching 1839 local vectors to 1828 remote vectors by collab.matchers.name_hash.NameHashMatcher yielded 1828 matches
    [2017-06-13 20:26:58,364: WARNING/Worker-5] Took: 0:00:00.230436
    [2017-06-13 20:26:58,464: WARNING/Worker-5] Matching 2075 local vectors to 2064 remote vectors by collab.matchers.assembly_hash.AssemblyHashMatcher yielded 2071 matches
    [2017-06-13 20:26:58,616: WARNING/Worker-5] Took: 0:00:00.250401
    [2017-06-13 20:26:58,623: INFO/MainProcess] Task collab.tasks.match[5f7446ed-667f-44a0-8363-6a1a88cfc3d2] succeeded in 3.52252092698s: None